lib/server-manager.js

```diff
@@ -35,19 +35,25 @@ export class ServerManager {
   _startForEditor: (editor: atom$TextEditor) => boolean;
   _startServer: (projectPath: string) => Promise<ActiveServer>;
   _changeWatchedFileFilter: (filePath: string) => boolean;
+  _getBusySignalService: () => ?atomIde$BusySignalService;
+  _languageServerName: string;
   _isStarted = false;
 
   constructor(
+    languageServerName: string,
```

Comment:
Perhaps move this parameter to the end of the list in case anyone is using this class directly.